install/update_91_92.php

```diff
@@ -800,31 +775,37 @@ function update91to92() {
       $DB->queryOrDie($query, "9.2 add saved searches alerts notification translation");
    }
 
+   // Yllen: must be done at the beginning for all tables to forget change place of the message if new table added in list
```

Comment:
I guess this should stay at the bottom, this is probably more "visible" for the users.

Anyways, I propose not to keep that kind of global message here, the Migration object can be in charge.

As example:
```diff
diff --git a/inc/migration.class.php b/inc/migration.class.php
index a8278f2..e2e74f2 100644
--- a/inc/migration.class.php
+++ b/inc/migration.class.php
@@ -51,6 +51,7 @@ class Migration {
       'pre'    => [],
       'post'   => []
    ];
+   private $has_backups;
 
    const PRE_QUERY = 'pre';
    const POST_QUERY = 'post';
@@ -647,6 +648,13 @@ class Migration {
          $DB->queryOrDie($query['query'], $query['message']);
       }
 
+      if ($this->has_backups) {
+         $migration->displayWarning(
+            __("You can delete backup tables if you have no need of them."),
+            true
+         );
+      }
+
       // end of global message
       $this->displayMessage(__('Task completed.'));
    }
@@ -854,7 +862,7 @@ class Migration {
     * @return boolean
     */
    public function backupTables($tables) {
-      $backup_tables = false;
+      $this->has_backups = false;
       foreach ($tables as $table) {
          // rename new tables if exists ?
          if (TableExists($table)) {
@@ -866,10 +874,10 @@ class Migration {
                   "backup_$table"
                )
             );
-            $backup_tables = true;
+            $this->has_backups = true;
             $this->renameTable("$table", "backup_$table");
          }
       }
-      return $backup_tables;
+      return $this->has_backups;
    }
 }
diff --git a/install/update_91_92.php b/install/update_91_92.php
index 299da48..0f49234 100644
--- a/install/update_91_92.php
+++ b/install/update_91_92.php
@@ -800,11 +800,6 @@ Regards,',
       $DB->queryOrDie($query, "9.2 add saved searches alerts notification translation");
    }
 
-   if ($has_backups) {
-      $migration->displayWarning("You can delete backup tables if you have no need of them.",
-                                 true);
-   }
-
    if (TableExists('glpi_queuedmails')) {
       $migration->renameTable("glpi_queuedmails", "glpi_queuednotifications");
       $migration->addPostQuery("UPDATE glpi_crontasks SET itemtype='QueuedNotification' WHERE itemtype='QueuedMail'");
```